main.go

```diff
@@ -376,3 +411,75 @@ func setupWebhooks(mgr ctrl.Manager) {
 		os.Exit(1)
 	}
 }
+
+// GetTLSOptionOverrideFuncs returns a list of TLS configuration overrides to be used
+// by the webhook server.
+func GetTLSOptionOverrideFuncs(options TLSOptions) ([]func(*tls.Config), error) {
```

Comment:
We can clean up this function a lot, or maybe even remove it, since there is no options to check for.

Reply:
I've cleaned up this function with only hardcoded tls values and also removed other code which is redundant now such as providing options to this function which are hardcoded inside it.